.gitignore

```diff
@@ -1,2 +1,3 @@
 /target
 Cargo.lock
+*.sqlite*
```

Comment:
This actually only concerns the oracle example so I would add a dedicated `.gitignore` file to the `rocket_sqlite_oracle` binary.